micom/workflows/build.py

```diff
@@ -130,7 +130,8 @@ def _summarize_models(args):
 
 
 def build_database(
-    manifest, out_path, rank="genus", threads=1, compress=None, progress=True
+    manifest, out_path, rank="genus", threads=1,
+    compress=False, compresslevel=6, progress=True,
```

Comment:
Probably should default to None.

```suggestion
    compress=None, compresslevel=6, progress=True,
```